pkg/authenticator/config/username_test.go

```diff
@@ -1,6 +1,8 @@
 package config
 
 import (
+	"fmt"
+
```

Comment:
Nit, maybe even not a problem... Do we need the space between the import of "fmt" and "testing"? They're both standard golang packages.

Reply:
Blah I merged before seeing this - you're right and I'll keep this in mind for future changes on this repo!